config/blockchain_network_config.go

```diff
@@ -0,0 +1,140 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	"io/ioutil"
+)
+
+type NetworkSelected struct {
+	NetworkName             string
+	EthereumJSONRPCEndpoint string
+	NetworkId               string
+	RegistryAddressKey      string
+}
+
+const (
+	DefaultBlockChainNetworkConfig string = `
+{
+  "local":{
+    "ethereum_json_rpc_endpoint":"http://localhost:8545",
+    "network_id":"999999",
+    "registry_address_key":"0x4e74fefa82e83e0964f0d9f53c68e03f7298a8b2"
+  },
+  "kovan":{
+    "ethereum_json_rpc_endpoint":"https://kovan.infura.io",
+    "network_id":"42"
+  },
+  "ropsten":{
+    "ethereum_json_rpc_endpoint":"https://ropsten.infura.io",
+    "network_id":"3"
+  },
+  "rinkeby":{
+    "ethereum_json_rpc_endpoint":"https://rinkeby.infura.io",
+    "network_id":"4"
+  },
+  "main":{
+    "ethereum_json_rpc_endpoint":"https://mainnet.infura.io",
+    "network_id":"1"
+  }
+}`
+	BlockChainNetworkFileName  = "blockchain_network_config.json"
+	EthereumJsonRpcEndpointKey = "ethereum_json_rpc_endpoint"
+	NetworkId                  = "network_id"
+	RegistryAddressKey         = "registry_address_key"
+	//Read this from a config file path todo
+	RegistryJsonFileName = "resources/blockchain/node_modules/singularitynet-platform-contracts/networks/Registry.json"
```

Comment:
Not sure why this file would be readable? It is a long filename, file as far as I understand should be inside NPM package. How will this path be available from daemon working directory?

Reply:
you are right , will fix this post Beta , closing this PR for now